utils/subset_vcf_for_phase.py

```diff
@@ -50,38 +27,41 @@ def get_subset_samples(
     """
     Get samples to subset out of v3 VDS.
 
-    :param samples_path: Path to TSV of sample IDs to subset to. The TSV must have a header of 's'.
-    :param pop: Optional str of genetic ancestry group to include in subset. Defaults to None.
-    :param hgdp: Boolean of whether to include HGDP in subset. Defaults to False.
-    :param tgp: Boolean of whether to include TGP in subset. Defaults to False.
+    :param samples_path: Path to TSV of sample IDs to subset to. The TSV must have a
+        header of 's'.
+    :param pop: Optional str of genetic ancestry group to include in subset. Default
+        is None.
+    :param hgdp: Boolean of whether to include HGDP in subset. Default is False.
+    :param tgp: Boolean of whether to include TGP in subset. Default is False.
     :return: Table of samples to subset to.
     """
     meta_ht = meta(data_type="genomes").ht()
     samples_to_keep = []
 
     def _add_filtered_meta(condition):
-        ht_to_add = meta_ht.filter(condition)
+        ht_to_add = meta_ht.filter(condition).select()
```

Comment:
```suggestion
        ht_to_add = meta_ht.filter(condition)
```
We do not want to remove all other columns here, we need them for export.